### PR TITLE
[Notifier] Add Sipgate bridge

### PR DESCRIPTION
**Pull request by sakul95**
Add a docs entry according to PR  [#sakul95:sipgate_notifier_transport](https://github.com/symfony/symfony/pull/57627)

### CODE REVIEW

File: notifier.rst [hunk 1]
```diff
@@ -159,6 +159,9 @@ Service
 `Sinch`_            **Install**: ``composer require symfony/sinch-notifier`` \
                     **DSN**: ``sinch://ACCOUNT_ID:AUTH_TOKEN@default?from=FROM`` \
                     **Webhook support**: No
+`Sipgate`_          **Install**: ``composer require symfony/sipgate-notifier`` \
```
Comment (OskarStark) [inline notifier.rst:162]: Please add a link at the end of the page, like done for the other bridges

+ add your bridge to the versionadded directive below this table, thanks

Comment (sakul95) [reply notifier.rst:162]: Sorry I dont see what you mean. A link to what? To the symfony repo where the bridge will be located? :)

Comment (OskarStark) [reply notifier.rst:162]: If you open this file and scroll to the bottom you will see it I guess 😃 

`Sipgate`_ means a link, and the link is defined down below

Comment (sakul95) [reply notifier.rst:162]: Ah sure thing, I got you, thanks! Now it's added.

**Comment by OskarStark** [comment]
Please add
```
.. versionadded:: 7.2

    The ``Sipgate`` integration was introduced in Symfony 7.2.
```

below the `versionadded:: 7.1`

Thanks

**Review by OskarStark — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you @sakul95.

**Pull request merged**